.github/workflows/ci.yml

```diff
@@ -96,7 +96,7 @@ jobs:
       # Any env name which does not start with `pyXY` will use this Python version.
       default_python: '3.11'
       envs: |
-        - linux: py311-test-oldestdeps-parallel-cov
+        - linux: py311-test-oldestdeps-cov
```

Comment:
Let's change this back.

Reply:
Done.

It's segfaulting when we run the tests serially too. That narrows it down a bit.